app/src/main/java/org/schabi/newpipe/player/helper/PlaybackParameterDialog.java

```diff
@@ -91,20 +99,25 @@ public double valueOf(final int progress) {
     double initialPitchPercent = DEFAULT_PITCH_PERCENT;
     @State
     boolean initialSkipSilence = DEFAULT_SKIP_SILENCE;
+    @State
+    boolean initialLiveEdge = DEFAULT_LIVE_EDGE;
```

Comment:
Oh, I just noticed, probably `initialiLiveEdge` is not needed. The other "initial" variables are for parameters that the dialog can change and that the dialog should be able to restore to their initial values when the user presses "Reset". But in this case "live edge" is not something the user can change, it is just a state of the player, so there is no need to store its initial value because it doesn't make sense to restore it, and also because the `liveEdge` variable will never change throughout the lifetime of the dialog.

Reply:
Ah, I overlooked that. I am not actually using the initialLiveEdge value anywhere. I will remove it.